source/about/deployments-and-editions.rst

```diff
@@ -1,53 +1,58 @@
+Team Edition is a free-to-use, open source, self-hosted collaboration platform offering all the core productivity benefits of competing SaaS solutions. It deploys as a single Linux binary with MySQL or PostgreSQL under an MIT license.
 
-To get started with a 14-day free trial of Mattermost Cloud online, visit https://customers.mattermost.com/cloud/signup. Signing up requires you to create an account with a business email and specify a domain for the workspace. Once you sign up and create your workspace, a wizard will help you get started completing your profile, inviting team members, and integrating with your favorite developer tools.
+Mattermost Team Edition is also bundled inside of the free Mattermost Enterprise Edition code base, which provides the same funcitionality as Mattermost Team Edition, with the additional benefit of being able to trial as well as upgrade into an expanded set of features available with paid subscription, such as "Mattermost Professional" and "Mattermost Enterprise". 
 
-Mattermost editions
---------------------
+Because of the benefits of Mattermost Enterprise Edition we recommend installing it over Mattermost Team Edition, even if you don’t currently need a subscription, so you'll have the flexibility to trial or enable additional features should you need them. However, if you only want to install software with a fully open source license, then Mattermost Team Edition is the best choice.
 
-The Mattermost platform is offered under four editions to meet the needs of organizations of all sizes.
+Mattermost Enterprise Edition 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Mattermost Enterprise Edition is the name for the binary of the Mattermost self-hosted Starter, Professional, and Enterprise plans. Mattermost Starter can be used for free without a subscription as commercial software that’s functionally equivalent to the open source Mattermost Team Edition licensed under MIT. When a subscription is purchased and applied to Mattermost Enterprise Edition, additional features unlock. The license agreement for Mattermost Enterprise Edition is included in the software and also available `here <https://mattermost.com/enterprise-edition-license/>`__.
+Our commercial, self-hosted software is called "Mattermost Enterprise Edition" and is available as a Linux Binary that deploys identically to our open source version--including upgrading in an identical fashion--with two key differences: a) It contains code for advanced commercial features, which can be unlocked with a valid license key, and b) it is offered under a commercial license (https://mattermost.com/enterprise-edition-license/). The commercial license prohibits reverse engineering and tampering with our license key mechanism unlocking paid features so that we can run a compliant and fair commercial business. 
```

Comment:
```suggestion
Our commercial, self-hosted software is called Mattermost Enterprise Edition, and it's available as a Linux Binary that deploys identically to our open source version -- including upgrading in an identical fashion -- with two key differences: a) It contains code for advanced commercial features, which can be unlocked with a valid license key; and b) it's offered under a commercial license (https://mattermost.com/enterprise-edition-license/). The commercial license prohibits reverse engineering and tampering with our license key mechanism unlocking paid features so that we can run a compliant and fair commercial business. 
```